infrastructure/template/api-policy.tmpl

```diff
@@ -0,0 +1,99 @@
+<policies>
```

Comment:
same as above, to the scope of this ticket, this will not change, why is this a diff here